docs/man/rpm-man-template.scd

```diff
@@ -0,0 +1,138 @@
+RPMCOMMAND(1)
+
+# NAME
+*rpmcmd* - Example rpm man page
+
+# SYNOPSIS
+Use the generic form for one-purpose commands:
+
+*rpmcmd* [options] [_ARGUMENT_] ...
+
+Use one entry per operation for multi-operation (aka mode) commands.
+Use {} for the operation switches:
+
+*rpmcmd* {*-b*|*--bardize*} [options] [bardize-options] [_ARGUMENT_] ...
+
+*rpmcmd* {*-f*|*--frobnize*} [options] [_ARGUMENT_] ...
+
+# DESCRIPTION
+Describe what *rpmcmd* command does in plain, understandable
+English. Command-line switches belong to *OPTIONS* and *OPERATIONS*
+as appropriate.
+
+Typographic conventions:
+- Use hard wrapping at 80 columns for all text
+- Use bold for literal values such as command name and options:
+*cmd* does X, with option *--do-this* does Y instead
+- Use bold when referring to section names
```

Comment:
I'd also add "uppercase", e.g. "Use bold and uppercase when referring to section names"

Reply:
More generally, use matching case - second level section names might not be all uppercase

Reply:
Added a note about the section case rule too.